.github/workflows/wide_integer.yml

```diff
@@ -7,6 +7,76 @@ on:
   schedule:
     - cron: '0 2 * * *' # run at 2 AM UTC
 jobs:
+  cmake-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [ g++, clang++ ]
+        include:
+          - compiler: g++
+            container: johnmcfarlane/cnl_ci:gcc-11
+          - compiler: clang++
+            container: johnmcfarlane/cnl_ci:clang-13-libcpp
+
+    runs-on: ubuntu-20.04
+    container: ${{matrix.container}}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: create build directory
+        run: mkdir $GITHUB_WORKSPACE/build
+
+      - name: build
+        working-directory: build
+        run: |
+          cmake \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DCMAKE_CXX_STANDARD=20 \
+              -DCMAKE_TOOLCHAIN_FILE=$GITHUB_WORKSPACE/.github/toolchains/gcc.cmake \
+              -GNinja \
```

Comment:
@ckormanyos `ninja` is a faster build tool than `make`. It also builds objects in parallel by default so build time for this job should be reduced quite a bit.